03_launch_mgmt_cluster.sh

```diff
@@ -195,6 +199,10 @@ IRONIC_FAST_TRACK=true
 RESTART_CONTAINER_CERTIFICATE_UPDATED="${RESTART_CONTAINER_CERTIFICATE_UPDATED}"
 EOF
 
+  if [ -n "${DEPLOY_ISO_URL}" ]; then
+    echo "DEPLOY_ISO_URL=${DEPLOY_ISO_URL}" | sudo tee -a "$IRONIC_DATA_DIR/ironic_bmo_configmap.env"
+  fi
+
```

Comment:
same here, we could probably reuse the tee we already do in https://github.com/metal3-io/metal3-dev-env/blob/08950cc47e6460de941f21229808fade04c7468e/03_launch_mgmt_cluster.sh#L183-L184